compiler/rustc_mir/src/dataflow/framework/graphviz.rs

```diff
@@ -570,6 +571,13 @@ where
     }
 }
 
+macro_rules! regex {
```

Comment:
I quite this. I think it would be a good addition to the `regex` crate (behind a feature for now). WDYT @hbina?

Reply:
I have opened an issue here https://github.com/rust-lang/regex/issues/709
I am still new to this so I haven't open a PR yet.